app/jobs/remote_server_job.rb

```diff
@@ -231,4 +138,24 @@ def ssh_exec!(ssh, command)
   def build_include_patterns(patterns)
     "-e \"INCLUDE_PATTERNS=#{patterns}\""
   end
+
+  def bare_metal_server_ip
+    Rails.application.secrets.bare_metal_server_ip
+  end
+
+  def bare_metal_server_user
+    Rails.application.secrets.bare_metal_server_user
+  end
+
+  def bare_metal_server_password
+    Rails.application.secrets.bare_metal_server_password
+  end
+
+  def api_name
+    Rails.application.secrets.api_name
+  end
+
+  def api_password
+    Rails.application.secrets.api_password
+  end
```

Comment:
@noahgibbs Do you think these 5 methods above are too much?

Maybe better way would be to make new module `Secrets` and move them there. That way whenever I need secrets I would include `Secrets` module.